src/Icon/Icon.jsx

```diff
@@ -15,20 +15,21 @@ export const Icon = ({ name, size, color }) => {
       : name
 
   return (
-    <StyledI color={color} size={size}>
+    <StyledI color={color} size={size} rotate={rotate}>
       {glyph}
     </StyledI>
   )
 }
 
-const StyledI = styled.i(({ color, size }) =>
+const StyledI = styled.i(({ color, size, rotate }) =>
   css({
     fontFamily: 'EtvasIcons',
     fontStyle: 'unset',
     fontWeight: 'normal',
     fontSize: size,
     lineHeight: size,
-    color
+    color,
+    animation: rotate ? 'rotation 2s infinite linear' : ''
```

Comment:
Maybe we can define in theme a default animation speed and use that variable?

Reply:
Ok